drivers/spi/spi_context.h

```diff
@@ -183,17 +203,44 @@ gpio_dt_flags_t spi_context_cs_active_level(struct spi_context *ctx)
 	return GPIO_ACTIVE_LOW;
 }
 
+static inline int spi_cs_configure(const struct spi_cs_control *cs_gpio)
+{
+	return gpio_pin_configure(cs_gpio->gpio_dev,
+			   cs_gpio->gpio_pin,
+			   cs_gpio->gpio_dt_flags |
+			   GPIO_OUTPUT_INACTIVE);
+}
+
+static int __attribute__((unused)) spi_context_cs_configure_all(struct spi_context *ctx)
```

Comment:
Function defined in a header should be `static inline`

Reply:
Fixed.